### PR TITLE
Fix merge bug

### PR DESCRIPTION
**Pull request by hayleyw7**
# Description drawing

- Fix merge bug with App and Error

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactor feature (non-breaking change that modifies existing work)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

- [x] local host 3000
- [ ] console.log()
- [ ] dev tools
- [ ] cypress testing

# Checklist:

- [x] My code follows the style guidelines of this project/linted
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by Matt-Roden — COMMENTED** [review]
Looks good!

File: src/components/App/App.js [hunk 1]
```diff
@@ -94,7 +94,7 @@ const App = () => {
               />
             )}
 
-            {error !== '' && (
+            {error !== '' && foundPokemon.length !== 0 && (
```
Comment (Matt-Roden) [inline src/components/App/App.js:111]: Nice!

**Review by Matt-Roden — APPROVED** [review]
thanks for this!

**Review by joanafbrito — APPROVED** [review]
Thank's for creating this

**Pull request merged**